signet.cabal

```diff
@@ -46,120 +46,99 @@ common executable
     -rtsopts
     -threaded
 
-common dependencies
+library
+  import: library
   build-depends:
     bytestring ^>=0.12,
     case-insensitive ^>=1.2.1.0,
     crypton ^>=0.34 || ^>=1.0.4,
     exceptions ^>=0.10.7,
-    memory ^>=0.18.0,
-    time ^>=1.12.2,
-
-library unstable
-  import: library, dependencies
-  build-depends:
     http-types ^>=0.12.4,
+    memory ^>=0.18.0,
+    tasty ^>=1.4 || ^>=1.5.3,
+    tasty-hunit ^>=0.10.2,
```

Comment:
I would like for the library to not depend on `tasty`, but this isn't actually a regression. The dependencies of library components are always required, even if the component isn't requested: https://github.com/haskell/cabal/issues/6039